### PR TITLE
Release drafter 5.14 - add autolabeler definition

### PR DESCRIPTION
**Pull request by warden**
Auto-label issues based on branches/titles/body.
https://github.com/release-drafter/release-drafter/releases/tag/v5.14.0

Ready for review: @jenkinsci/github-admins

### CODE REVIEW

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -79,3 +79,21 @@ replacers:
     replace: 'Custom WAR Packager'
   - search: '@dependabot-preview'
     replace: '@dependabot'
+
+autolabeler:
+  - label: 'chore'
```
Comment (oleg-nenashev) [inline .github/release-drafter.yml:84]:
```suggestion
  - label: 'documentation'
```

Maybe Asciidoc as well

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Looks like one label is wrong

**Review by olblak — APPROVED** [review]

File: .github/release-drafter.yml (hunk 1 above)
Comment (warden) [reply .github/release-drafter.yml]: done, asciidoc is not present in the categories at the moment

**Review by oleg-nenashev — APPROVED** [review]
One thing is that conditions are added with "OR", so any match is enough. It means that enhancement pull requests with documentation are likely to set two labels and then lead to two changelog entries. Maybe not the best behavior for the 'files' filter.

CC @jetersen

**Review by olblak — APPROVED** [review]

**Comment by timja** [comment]
🚢 ?

**Comment by oleg-nenashev** [comment]
I think we should extend permissions in this repo. Sorry for missing so many pull requests

**Pull request merged**